app/javascript/packs/software_versions/software_versions_template.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="col-md-7 col-lg-8 mb-4 pb-2 mt-4">
+    <div class="card">
+      <div class="card-content pb-0">
+        <h2 class="h4 card-heading">Software Versions</h2>
+        <div v-if="this.is_loading" class="text-center text-muted pb-4">
+          loading ...
+        </div>
+      </div>
+
+      <table class="table" v-if="!this.is_loading">
+        <thead>
+        <tr>
+          <th>Software Version</th>
+          <th>Stake %</th>
+          <th>No. of validators</th>
+        </tr>
+        </thead>
+        <tbody>
+          <tr v-for="version in Object.keys(software_versions).slice(0, 6)" :key="version">
+            <td :class="version == current_software_version ? 'h4 text-success' : '' " >
```

Comment:
`version == current_software_version ? 'h4 text-success' : '' `

We can use a helper for that. Especially, since this is being used more times below.